packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -466,9 +466,7 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
       [size]: tabMeta ? tabMeta[size] : 0,
     };
 
-    // IE11 support, replace with Number.isNaN
-    // eslint-disable-next-line no-restricted-globals
-    if (isNaN(indicatorStyle[startIndicator]) || isNaN(indicatorStyle[size])) {
+    if (Number.isNaN(+indicatorStyle[startIndicator]) || Number.isNaN(+indicatorStyle[size])) {
```

Comment:
Actually we don't need to check for `NaN` global type since it will be never be `NaN`. See discussion in #41709 especially https://github.com/mui/material-ui/pull/41709#issuecomment-2044685175 and https://github.com/mui/material-ui/pull/41709#issuecomment-2046619325. We use a `typeof` number check instead.

Reply:
Thanks for bringing this up, updated!